src/ansiblelint/rules/meta_runtime.py

```diff
@@ -32,8 +32,9 @@ class CheckRequiresAnsibleVersion(AnsibleLintRule):
     # Refer to https://access.redhat.com/support/policy/updates/ansible-automation-platform
     # Also add devel to this list
     supported_ansible = ["2.14.", "2.15.", "2.16."]
+    supported_ansible_examples = [f">={x}0" for x in supported_ansible]
     _ids = {
-        "meta-runtime[unsupported-version]": "'requires_ansible' key must be set to a supported version - 2.13.x, 2.14.x, 2.15.x",
+        "meta-runtime[unsupported-version]": f"'requires_ansible' key must refer to a currently supported version such: {', '.join(supported_ansible_examples)}",
```

Comment:
This isn't necessarily the cleanest way of doing this, but I think the _.x_ part is important to clarifying what we're expecting here.
```suggestion
        "meta-runtime[unsupported-version]": f"'requires_ansible' key must refer to a currently supported version such as: {', '.join((v + "x" for v in supported_ansible)}",
```